### ISSUE TITLE
warm_up

### ISSUE THREAD

**Issue by Sunshine19980919** [layumi/Person_reID_baseline_pytorch#338]
Hello, thank you for your excellent work

I found that in the warm up strategy, you generally deal with the learning rate, but you choose to deal with losses. Can you explain the reason for doing so? Or can you provide me with paper materials? I'm just confused.

Once again, I salute your work.

**Comment by layumi** [comment]
Hi @Sunshine19980919 
They are very close actually. I just found loss warmup is easier to write the code. 

You may take care of weight decay. This might be the only difference.

**Comment by Sunshine19980919** [comment]
All right, thank you very much for your answer!
